### DISCUSSION TITLE
Error AttributeError: 'res.users' object has no attribute 'chatter_position'

### DISCUSSION THREAD

**Discussion by it12uw** [OCA/odoo-community.org#106] [Q&A]
hello , i have problem with my odoo14, so yesterday i try to install backend theme from odoo app store and surprisely the installation process is not completed, so i refresh the browser and when i try to access the localhost : 8609 it give me the 500 Internal Server Error.
and when i see the log file, theres a error message **AttributeError: 'res.users' object has no attribute 'chatter_position'
Error to render compiling AST
AttributeError: 'res.users' object has no attribute 'chatter_position'
Template: web.webclient_bootstrap
Path: /t/t/t[5]
Node: <t t-set="body_classname" t-value="'o_web_client' + ' o_chatter_position_' + (request.env.user.chatter_position or 'normal')"/>**

i try so many ways to fix this, even i try to uninstaled the module from odoo shell but still not working.

any idea how to fix this problem?
so i can access the localhost : 8069 again.

**Comment by chienandalu** [comment]
Looks like the template web.webclient_bootstrap is broken by that theme. If
you can open a shell or if you have direct write access to your db you can
restore it manually from the base code and hopefully that would let you go
to apps and uninstall the module.

**Reply by it12uw** [reply]
yes, i have direct access to the database, but i donw know how to do it.
